src/mmw/js/src/modeling/models.js

```diff
@@ -103,8 +136,8 @@ var ProjectModel = Backbone.Model.extend({
 
 var ModificationModel = coreModels.GeoModel.extend({
     defaults: _.extend({
-            name: '',
-            type: ''
+            'name': '',
+            'type': ''
```

Comment:
Ok. I'll remove them. My editor was highlighting `type` so I quoted it to reduce visual noise.